CMakeLists.txt

```diff
@@ -10,6 +10,12 @@ if (POLICY CMP0072)
     set(OpenGL_GL_PREFERENCE LEGACY)
 endif(POLICY CMP0072)
 
+if (BUILD_WITH_CONDA AND WIN32)
```

Comment:
BUILD_WITH_CONDA is used now before it's defined in InitializeFreeCADBuildOptions.cmake. What about writing:
```
if (WIN32)
    option(FREECAD_USE_CCACHE "Auto detect and use ccache during compilation" OFF)
else()
    option(FREECAD_USE_CCACHE "Auto detect and use ccache during compilation" ON)
endif()
```
?

Reply:
BUILD_WITH_CONDA has to be passed as an argument to cmake AFAIK so it should exist when relevant I think, I'll check again.

But ccache is not very effective on windows anyways, at least not when using msvc so we could disable it by default on windows, than if someone really wants to use it they can enable it.